shepherd_bms/src/calcs.cpp

```diff
@@ -30,17 +30,21 @@ void calcPackTemps(AccumulatorData_t *bmsdata)
     int totalTemp = 0;
     for(uint8_t c = 1; c < NUM_CHIPS; c += 2)
     {
-        for(uint8_t therm = 17; therm < 28; therm++) {
-            if (bmsdata->chipData[c].thermistorValue[therm] > bmsdata->maxTemp.val) {
+        for(uint8_t therm = 17; therm < 28; therm++) 
+        {
+            // finds out the maximum cell temp and location
+            if (bmsdata->chipData[c].thermistorValue[therm] > bmsdata->maxTemp.val) 
                 bmsdata->maxTemp = {bmsdata->chipData[c].thermistorValue[therm], c, therm};
-            }
-            if (bmsdata->chipData[c].thermistorValue[therm] < bmsdata->minTemp.val) {
+
+            // finds out the minimum cell temp and location
+            if (bmsdata->chipData[c].thermistorValue[therm] < bmsdata->minTemp.val) 
```

Comment:
ok this is picky but ive never been a fan of no bracket one line if statements which i think is what u did here

Reply:
but as long as it was intentional to make it look neater no big deal

Reply:
My argument for no brackets on one-line ifs is that it makes it easier to read, especially if you have multiple checks like we have for the chargingCheck and the balancingCheck. It kinda reads like a sentence and removes a lot of clutter

Reply:
I'm deffo on team "always use brackets". When I'm reading code Idrc if it reads like a sentence, I just want it to be predictable

Reply:
Yeah I'm sorta in the middle, but could totally see bugs being more of an issue with no brackets. If I had to lean one way it would be pro-brackets. Slightly more cluttered > obvious source of bugs imo